app/ide-desktop/lib/dashboard/src/layouts/DriveBar.tsx

```diff
@@ -75,24 +90,58 @@ export default function DriveBar(props: DriveBarProps) {
     })
   }, [isCloud, doCreateDirectory, doCreateProject, /* should never change */ inputBindings])
 
+  const searchBar = (
+    <AssetSearchBar
+      backend={backend}
+      isCloud={isCloud}
+      query={query}
+      setQuery={setQuery}
+      suggestions={suggestions}
+    />
+  )
+  const assetPanelToggle = (
+    <>
+      {/* Spacing. */}
+      <div
+        className={tailwindMerge.twMerge(
+          'transition-all duration-side-panel',
+          !isAssetPanelOpen && 'w-5'
+        )}
+      />
+      <div className="absolute right-[15px] top-[25px] z-1 transition-all duration-side-panel">
+        <ariaComponents.Button
+          size="medium"
+          variant="custom"
+          icon={RightPanelIcon}
+          aria-label={isAssetPanelOpen ? getText('openAssetPanel') : getText('closeAssetPanel')}
+          className={tailwindMerge.twMerge('selectable', isAssetPanelOpen && 'active')}
```

Comment:
> it doesn't know what `active` is

i don't think it hurts even though this is the case, since if we *don't* use tw-merge here and then add a tailwind class then it will be wrong

Reply:
The main issue with "active" is that it breaks class deduplication, which leads to CSS specificity issues.
If you want to hide a set of reusable classes behind a "simple" abstraction, you can choose from these options:
1. Introduce a variable: `const ACTIVE = 'bg-white ...'`
2. Introduce a new variant (this might be helpful if you want to add customization/variants to it)